### ISSUE TITLE
auth not working with .authinfo or .authinfo.gpg

### ISSUE THREAD

**Issue by mantlepro** [hexmode/mediawiki-el#13]
mediawiki.el not using .authinfo and .authinfo.gpg to authenticate for empty passwords.

From the README:

 > If mediawiki sees that the password is empty, then it will use the auth-source library to retrieve the password. This will allow you to use encryption to store your passwords.

    #~/.emacs
    '(mediawiki-site-alist
       (quote(("mywiki" "https://mywiki.fqdn/" "myuser" "" "local" "Main Page")))
       )

    #~/.authinfo.gpg
    machine mywiki.fqdn login myuser password "mypassword"

Is this a properly formatted example for mediawiki.el to accept it? I've also tried adding https (443) to  '(auth-source-protocols)

Tested with both .authinfo.gpg and .authinfo with emacs properly configured (e.g. authinfo.gpg works for tramp, etc.). I also have the following in my .emacs file:

    #~/.emacs
    (require 'auth-source)
    (require 'mediawiki)

Using version 20160902.827 (melpa). Anyway, thanks for the excellent mode. Glad to be editing wikipedia pages in emacs.

**Comment by hexmode** [comment]
The format url-http uses for finding the authinfo stuff is:
```
machine http://mywiki.fqdn/ login myuser password "mypassword"
```
Note that _machine_ needs to match whatever is given as your url.

**Comment by hexmode** [comment]
reopening.  it isn't working.

**Comment by hexmode** [comment]
fixed in 692490eef73f3521410385e4a8ad6d058bbac726
